antora-ui-camel/src/css/frontpage.css

```diff
@@ -114,6 +114,12 @@ section.frontpage.columns.blog-posts {
   background-color: var(--color-smoke-50);
 }
 
+section.frontpage.blog-posts ul {
+  display: flex;
```

Comment:
```suggestion
  display: flex;
  flex-wrap: wrap;
```

Reply:
This change seems problematic.. This is how it looks for me:

![Screenshot 2020-08-12 at 1 27 16 PM](https://user-images.githubusercontent.com/32356795/89990275-bdc5f780-dc9f-11ea-897a-2eeab6a2326b.png)

I've suggested another change to make sure the **flex-direction** of the **blog-posts** section becomes **'column'** for <1023px.